src/statisticItem.js

```diff
@@ -160,9 +185,12 @@ export default class StatisticItem extends Component {
       kpisRows,
       isShow
     };
-    const icon = (
-      <Icon isOnValue={true} valueIcon={valueIcon} iconSize={iconSize} value={numericValue} infographic={infographic} />
-    );
+    const icon = () => {
+      return (
+        <Icon ref="icon" columnNumber={columnNumber} isOnValue={true} valueIcon={valueIcon} iconSize={iconSize} value={numericValue} infographic={infographic} />
```

Comment:
Why do we have check `isOnValue` inside the Icon component if it's set `true` here?

Reply:
because in other cases, we send that prop as false/don't send it at all

Reply:
okay, I see